docs/tutorial/population_dynamics.rst

```diff
@@ -124,55 +173,82 @@ bins to be plotted as `n_bins`.
+The `alpha` argument determines the CI that is estimated as `100*(1-alpha)` % CI; in this case we're showing the estimate for the 95% CI.
+The values for the lower and upper bounds are the `100*(alpha/2)` and `100*(1-alpha/2)` percentiles of the distributions.
 
-Note, however, that while the means correspond well with the optimized model parameters, the distributions are not symmetric.
-In such a case, the simple method of using percentiles as CI values may not be appropriate. For more advanced analysis,
-the distribution values are directly available through the properties `DwelltimeModel.bootstrap.amplitude_distributions` and
-`DwelltimeModel.bootstrap.lifetime_distributions` which return the data as a `numpy` array with
-shape `[# components, # bootstrap samples]`.
+Clearly the distributions here are not Gaussian. Specifically, the two distributions on the left for the fractional amplitudes
+are split, indicating that the model parameters are non-identifiable (there are different combinations of parameters that give
+similar overall results). This analysis strongly indicates that the single exponential model is preferable. We can also look at
+the bootstrap for that model to verify the results are satisfactory::
+
+    dwell_1.calculate_bootstrap(iterations=100)
+
+    plt.figure()
+    dwell_1.bootstrap.plot(alpha=0.05)
+    plt.show()
+
+.. image:: figures/population_dynamics/dwell1_bootstrap.png
+
+Here we only see one distribution since the fractional amplitude for a single exponential model is `1` by definition. The results
+look much better, with most of the distribution being fairly Gaussian with the exception of some outliers at longer lifetimes.
+These likely are the result of poorly fit or numerical unstable models.
+
+.. note::
+    As we have seen, care must be taken when choosing between different models and interpreting the bootstrapped confidence intervals.
+    The means of the bootstrap distribution should correspond well with the optimized model parameters from the original data.
+    Here, we only ran 100 iterations of the sampling to keep the analysis time short for exploratory. When computing the distributions
+    to obtain final values when a model has been selected, you should generally run significantly more iterations.
+
+    As mentioned before, ideally the bootstrapped distributions should also be normally distributed. However sometimes even
+    valid models can yield skewed distributions. In such a case, the simple method of using percentiles as CI values may not be appropriate.
+    For more advanced analysis, the distribution values are directly available through the properties
+    :attr:`~lumicks.pylake.population.dwelltime.DwelltimeBootstrap.amplitude_distributions`
+    and :attr:`~lumicks.pylake.population.dwelltime.DwelltimeBootstrap.lifetime_distributions` which return the data as a `numpy` array with
+    shape `[# components, # bootstrap samples]`.
```

Comment:
When reading the next section about the limitations, new reaction partners and their concentrations feel somehow to "come out of the blue" as the rates for the unfolding and folding of the hairpin do not depend on the concentration, but rather on the applied force. Would be nice for the future to have a little transition/introductory sentence in the next section that relates the limitations to this section and points out the difference.

Reply:
hmm yeah, because this all is really first order kinetics and at the single molecule level concentration essentially becomes `bool` 😆 I'll add something, and maybe even up at the top some examples of common data you would want to analyze with dwell time analysis (this stuff, binding, etc...) just to prime the reader to think about different kinds of experiments